docs/user-guide/TutorialDemoServer.md

```diff
@@ -4,13 +4,13 @@ Let's try RESTler on a simple example of REST API and service.
 
 The directory `restler\demo_server` in this repo contains a self-contained example of service you can run locally on your machine. See the [README.md](/demo_server/README.md) file in that directory to install and start this demo_server service.
 
-Let's create a fresh directory `C:\restler-test` where will run RESTler on this example. Let's cd into that dir: `cd C:\restler-test`
+Let's create a fresh directory `C:\demo-server-test` where will run RESTler on this example. Let's cd into that dir: `cd C:\demo-server-test`. Make sure your demo server is running before proceeding with the next part.
```

Comment:
>proceeding with the next part. [](http://example.com/codeflow?start=188&length=30)

Now that I read this again, can you please move this sentence to the end of the previous paragraph?  Then we can say "make sure your demo server is running before starting this tutorial".